src/gmp/models/credential.ts

```diff
@@ -185,6 +192,17 @@ class Credential extends Model {
 
     ret.allow_insecure = parseYesNo(element.allow_insecure);
 
+    if (element.type === KRB5_CREDENTIAL_TYPE) {
+      const kdcsRaw = (element as CredentialElement).kdcs;
```

Comment:
Thought: I don't think we need the casting, also on line 203.
```suggestion
      const kdcsRaw = (element).kdcs;
```